armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -480,7 +481,9 @@ def _checkComponentConsistency(b, repBlock):
 
             theseNucs = set(c.getNuclides())
             thoseNucs = set(repC.getNuclides())
-            diffNucs = theseNucs.symmetric_difference(thoseNucs)
+            diffNucs = theseNucs.symmetric_difference(thoseNucs).intersection(
+                consistentNucs
+            )
```

Comment:
This check is, IMO, pretty opaque and could use a comment. It works, but is one of those "elegant but unreadable" pieces of code.